lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -311,7 +311,11 @@ def execute
         end
 
         json = export_json(export_json_options)
-        create_tar(cv, cvv, repositories, json)
+        if repositories&.any? or cv['composite']
```

Comment:
Use `||` instead of `or`, due to [precedent issues](http://ruby-doc.org/core-2.6.3/doc/syntax/precedence_rdoc.html). I'm surprised rubocop didn't complain about this.

Reply:
rubocop has complained about it. I just thought to correct it later, to avoid unnecessary commit.